src/address.rs

```diff
@@ -136,13 +161,10 @@ impl<A, Rc: RefCounter> Address<A, Rc> {
 
     /// Convert this address into a generic address which can be weak or strong.
     pub fn as_either(&self) -> Address<A, Either> {
-        Address {
-            sink: self.sink.clone(),
-            ref_counter: self.ref_counter.clone().into_either(),
-        }
+        Address(self.0.clone().into_either_rc())
     }
 
-    /// Send a message to the actor.
+    /// Send a message to the actor with a given priority.
```

Comment:
This is no longer taking a priority as an argument so it is a bit confusing here I think.

Reply:
Good point, copy paste error on my part